### PR TITLE
feat: add afrikaans locale

### PR DESCRIPTION
**Pull request by hansieodendaal**
Description
---
Added Afrikaans locale

Motivation and Context
---
To expand the locale

How Has This Been Tested?
---
Checked against commonly used terms

What process can a PR reviewer use to test or verify this change?
---
Test against common Afrikaans

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: public/locales/af/common.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "submit": "Indien",
```
Comment (SWvheerden) [inline public/locales/af/common.json:2]:
```suggestion
    "submit": "Dien in",
```
I think its the other way around

File: public/locales/af/common.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "cancel": "Kanselleer",
+    "temperature": "Temperatuur",
+    "max-temperature": "Maksimum temperatuur",
+    "usage": "Gebruik",
+    "seconds": "sekondes",
```
Comment (SWvheerden) [inline public/locales/af/common.json:7]:
```suggestion
    "seconds": "Sekondes",
```

File: public/locales/af/mining-view.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "floor": "Vloer",
+    "current-block-time": "Huidige bloktyd",
+    "your-reward-is": "Jou beloning is",
+    "connection-to-node-lost": "Mynwerker verbinding verloor. Wag asseblief vir mynwerker verbinding of begin weer.",
```
Comment (SWvheerden) [inline public/locales/af/mining-view.json:5]:
```suggestion
    "connection-to-node-lost": "Mynwerker het verbinding verloor. Wag asseblief vir mynwerker om verbinding terug te kry of begin weer.",
```

Comment (SWvheerden) [inline public/locales/af/mining-view.json:5]: or I would put it completely different and a more direct translation to
```suggestion
    "connection-to-node-lost": "Kontak met mynwerker verloor. Wag asseblief vir mynwerker om kontak te maak of begin weer.",
```

File: public/locales/af/settings.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+    "settings": "Instellings",
+    "seed-words": "Saadwoorde",
+    "change-language": "Verander taal",
+    "logs": "Verslae",
+    "open-logs-directory": "Maak verslag vouer oop",
```
Comment (SWvheerden) [inline public/locales/af/settings.json:6]:
```suggestion
    "open-logs-directory": "Maak verslag lêr oop",
```

File: public/locales/af/mining-view.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    "your-reward-is": "Jou beloning is",
+    "connection-to-node-lost": "Kontak met mynwerker verloor. Wag asseblief vir mynwerker om kontak te maak of begin weer.",
+    "mining-button-text": {
+        "starting-mining": "Besig om te Begin Myn",
+        "start-mining": "Begin Myn",
+        "pause-mining": "Onderbreek Myn",
+        "changing-mode": "Verander Modus",
+        "cancel-mining": "Kanselleer Myn",
+        "waiting-for-idle": "Wag tot Onaktief",
+        "started-auto-mining": "Outomatiese Myn het Begin"
```
Comment (SWvheerden) [inline public/locales/af/mining-view.json:13]:
```suggestion
        "starting-mining": "Myn begin",
        "start-mining": "Begin myn",
        "pause-mining": "Onderbreek myn",
        "changing-mode": "Verander modus",
        "cancel-mining": "Kanselleer myn",
        "waiting-for-idle": "Wag tot onaktief",
        "started-auto-mining": "Outomatiese myn het begin"
```

File: public/locales/af/settings.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+    "settings": "Instellings",
+    "seed-words": "Saadwoorde",
+    "change-language": "Verander taal",
+    "logs": "Verslae",
+    "open-logs-directory": "Maak verslag lêr oop",
```
Comment (SWvheerden) [inline public/locales/af/settings.json:6]:
```suggestion
    "open-logs-directory": "Maak verslag lêer oop",
```

File: public/locales/af/sidebar.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+    "auto-miner": "Outomatiese Mynwerker",
+    "auto-miner-description": "Outomatiese mynwerker sal aanskakel wanneer jou masjien onaktief is",
+    "next-milestone": "Volgende Mylpaal",
+    "wallet-balance": "Beursie Balans",
+    "setup-scheduler": "Stel Skeduleerder op",
```
Comment (SWvheerden) [inline public/locales/af/sidebar.json:6]:
```suggestion
    "auto-miner": "Outomatiese mynwerker",
    "auto-miner-description": "Outomatiese mynwerker sal aanskakel wanneer jou masjien onaktief is",
    "next-milestone": "Volgende mylpaal",
    "wallet-balance": "Beursie balans",
    "setup-scheduler": "Stel skeduleerder op",
```

File: public/locales/af/setup-view.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
+{
+    "setting-up": "Stel Tari waarheids masjien op...",
```
Comment (SWvheerden) [inline public/locales/af/setup-view.json:2]:
```suggestion
    "setting-up": "Stel Tari waarheidsmasjien op...",
```

**Review by stringhandler — APPROVED** [review]

**Pull request merged**